compose/nginx_site.conf

```diff
@@ -1,8 +1,17 @@
 server {
+    listen 80;
     index index.php;
     server_name php-docker.local;
+    error_log  /var/log/nginx/error.log;
+    access_log /var/log/nginx/access.log;
```

Comment:
are access logs necessary? what about /dev/null instead?

Reply:
should also be possible

Reply:
We run nginx in the foreground so don't those logs default to std out anyway?

Reply:
mh - i'm not totally sure atm, but i think you're right - but then we can remove the logs complete?

Reply:
i have removed the logging paths, because its running in foreground of the docker container and is printed also to the stdout where a user can fetch the logs